content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -136,6 +136,33 @@ The following table shows the required settings for many common Kubernetes envir
 | Red Hat OpenShift 3.10+ | _(none)_ | _(none)_ |
 | Red Hat OpenShift 4.2+ | `--set components.cni.namespace=kube-system --set values.cni.cniBinDir=/var/lib/cni/bin --set values.cni.cniConfDir=/etc/cni/multus/net.d --set values.cni.chained=false --set values.cni.cniConfFileName="istio-cni.conf" --set values.sidecarInjectorWebhook.injectedAnnotations."k8s\.v1\.cni\.cncf\.io/networks"=istio-cni` | _(none)_ |
 
+#### Istioctl support in 1.4.8+ for OpenShift 4.2+
+
+Istioctl in 1.4.8+ does not support [escaping strings](https://github.com/istio/istio/issues/19196). To install on
+OpenShift 4.2+ you will need to create a file and pass that into istioctl:
+
+cni-annotations.yaml
+```yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+  values:
+    cni:
+      chained: false
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      cniConfFileName: istio-cni.conf
+    sidecarInjectorWebhook:
+      injectedAnnotations:
+        "k8s.v1.cni.cncf.io/networks": istio-cni
+```
+
+Then passing in the file in as an argument `-f cni-annotations.yaml`
```

Comment:
I'm making an assumption that people want to add more to their command lines than just this flag, else you could use a text bash to make something like `istioctl manifest generate -f cni-annotations.yaml` or whatever form the command is to simplify cut/paste.

Reply:
Hopefully I made it more clear that other variables can be set or changed when doing an install. Need further review.